config/jobs/kubernetes-sigs/cluster-api/cluster-api-main-presubmits.yaml

```
@@ -411,3 +411,55 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api
       testgrid-tab-name: capi-pr-e2e-conformance-ci-latest-main
+
+  - name: pull-cluster-api-e2e-latestk8s-1-31-1-32-main
```

```suggestion
  - name: pull-cluster-api-e2e-latestk8s-main
```
We have to update the template so this looks like ^^

---

updated!